analysis_options.yaml

```diff
@@ -1,14 +1,12 @@
 include: package:flutter_lints/flutter.yaml
 
 analyzer:
-  errors:
-    invalid_annotation_target: ignore
-    deprecated_member_use_from_same_package: ignore
   exclude:
     - lib/models/*.freezed.dart
     - lib/models/*.g.dart
     - lib/models/**/*.freezed.dart
     - lib/models/**/*.g.dart
+    - api_tester/**
```

Comment:
The api_tester already runs the linter separately as part of the api tests. It was conflicting with the new version of flutter_lints if I didn't do this.